src/test/java/io/antmedia/test/MuxerUnitTest.java

```diff
@@ -1282,4 +1271,117 @@ public AppSettings getAppSettings() {
 		return appSettings;
 	}
 
+	@Test
+	public void testRecording(){
+		testRecording("dasss",true);
+	}
+
+	public void testRecording(String name, boolean checkDuration){
```

Comment:
Thank you for test code. We need to add test code for EncoderAdaptor in Enterprise project.